src/web/app/components/comment-box/comment-row/comment-row.component.html

```diff
@@ -33,9 +33,9 @@
         <i class="fa fa-eye-slash" aria-hidden="true" ngbTooltip="This response comment is visible to {{ visibilityStateMachine.getVisibilityTypesUnderVisibilityControl(CommentVisibilityControl.SHOW_COMMENT) | commentVisibilityTypesJointName }}"></i>
 
         <div class="float-right comment-row">
-          <button type="button" class="btn btn-secondary btn-sm" *ngIf="!shouldHideEditButton" ngbTooltip='Edit this comment'
+          <button type="button" class="btn btn-secondary btn-sm" *ngIf="!shouldHideEditButton" ngbTooltip='Edit'
                   (click)="triggerModelChange('isEditing', true)" [disabled]="isDisabled"><i class="fas fa-pencil-alt"></i></button>
-          <button type="button" class="btn btn-secondary btn-sm" *ngIf="!shouldHideDeleteButton" ngbTooltip='Delete this comment'
+          <button type="button" class="btn btn-secondary btn-sm" *ngIf="!shouldHideDeleteButton" ngbTooltip='Delete'
```

Comment:
Do you even need a tooltip here actually? Or is the tooltip the name of the button?

Reply:
I think the tooltip here should remain as before, otherwise just "Edit" and "Delete" are as good as no tooltip if the icons are representative enough of the actions

Reply:
I wanted a tooltip here just in case the user fails to recognize the icon for some reason.

> I think the tooltip here should remain as before, otherwise just "Edit" and "Delete" are as good as no tooltip if the icons are representative enough of the actions

Yup will revert the tooltip then.